core/src/main/java/io/ddf/datasource/DataFormat.java

```diff
@@ -1,7 +1,7 @@
 package io.ddf.datasource;
 
 public enum DataFormat {
-  UNDEF, SQL, CSV, TSV, JSON, PQT;
+  UNDEF, SQL, CSV, TSV, JSON, PQT, AVRO, ORC;
```

Comment:
We should never use enum ordinals in a switch statements. If the symbols are re-ordered then the fromInt method will break.
Please add the integer value as an explicit member of the enum.